docs/changelog/109925.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 109925
+summary: Use a `xpack.stack.template.index.mode` variable to enable LogsDB in `logs@settings`
```

Comment:
Should we add something around logs to the name as this only applies to the logs template?

Reply:
Like `xpack.stack.template.logs.index.mode`? Or what?

Reply:
That sgtm. I wonder if we need to add a changeling for this, though. It's not really something user-facing and I don't think there has been a release, so I wouldn't consider this to be a `>bug`. If you agree, you can tag this as `>non-issue`, which also gets rid of the requirement to have a changelog.